rln/Cargo.toml

```diff
@@ -33,6 +33,7 @@ num-bigint = { version = "0.4.3", default-features = false, features = ["rand"]
 num-traits = "0.2.11"
 once_cell = "1.14.0"
 rand = "0.8"
+rand_chacha = "0.3.1"
```

Comment:
I am worried about the new dependency here. How does it affect the wasm blob size? I intended to add some size checking as part of the CI but it was not as straightforward as I hoped because cargo make is used instead of npm.

Cc @richard-ramos

Reply:
Right now building this branch, i end up with 530KB which is way better than the 838KB of the version published https://unpkg.com/@waku/rln@0.0.8-964df01/bundle/assets/rln_wasm_bg-0185b546.wasm
```
-rw-rw-r-- 1 richard richard  17K sep 24 12:15 rln_wasm_bg.js
-rw-rw-r-- 1 richard richard 530K sep 30 08:42 rln_wasm_bg.wasm
-rw-rw-r-- 1 richard richard 1.1K sep 30 08:42 rln_wasm_bg.wasm.d.ts
-rw-rw-r-- 1 richard richard 3.2K sep 30 08:42 rln_wasm.d.ts
-rw-rw-r-- 1 richard richard  20K sep 30 08:42 rln_wasm.js
```

Reply:
What changed ???